contrib/babelfishpg_tsql/src/pltsql_coerce.c

```diff
@@ -371,6 +372,32 @@ tsql_precedence_info_t tsql_precedence_infos[] =
 
 #define TOTAL_TSQL_PRECEDENCE_COUNT (sizeof(tsql_precedence_infos)/sizeof(tsql_precedence_infos[0]))
 
+/* Following constants value are defined based on the exception function list */
+#define EFUNC_MAX_ARGS 2
+#define EFUNC_MAX_VALID_TYPES 4
+
+typedef struct tsql_valid_type
+{
+	int		len;
+	char   *valid_types[EFUNC_MAX_VALID_TYPES];
+	Oid		valid_types_oid[EFUNC_MAX_VALID_TYPES];
```

Comment:
Why are we maintaining OIDs? Isn't type_names will be sufficient?

Reply:
removed maintaining OID.